### PR TITLE
Task connect db into Story pull request

### PR DESCRIPTION
**Pull request by ChrisTregaskis**

### CODE REVIEW

File: Resources/collectionApp_2020-02-05.sql [hunk 1]
```diff
@@ -20,6 +20,31 @@
...
+/*!40000 ALTER TABLE `cheese` DISABLE KEYS */;
+
+INSERT INTO `cheese` (`id`, `Name`, `Country of origin`, `Wine pairing`, `Fun fact`)
+VALUES
+	(1,'Stilton','England','Lambrusco Dolce','Must be produced in Derbyshire, Leicestershire or Nottinghamshire.');
+
+/*!40000 ALTER TABLE `cheese` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
```
Comment (daniellaskinner) [inline Resources/collectionApp_2020-02-05.sql]: make sure no cheese info in your project!

File: Resources/collectionApp_2020-02-05.sql [hunk 2]
```diff
@@ -7,7 +7,7 @@
 #
 # Host: 127.0.0.1 (MySQL 5.7.29)
 # Database: collectionApp
```
Comment (daniellaskinner) [inline Resources/collectionApp_2020-02-05.sql]: rename your db chris_wine_collection so its clear for Charlie and make sure to update this in your PDO connection too

File: index.php [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
 
+require_once 'functions.php';
+require_once 'connectDB.php';
 
+$db = connectDB();
+$wines = getWines($db);
 
-
-
+$displayWines = '';
```
Comment (daniellaskinner) [inline index.php:9]: you can delete this line as $displayWines gets defined on line 11, i think..

File: styles.css [hunk 4]
```diff
@@ -130,3 +131,14 @@ li span {
...
 
+.view-more-link {
+    width: 230px;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+    padding: 2px;
+    border-bottom: 1.4px solid black;
```
Comment (daniellaskinner) [inline styles.css:142]: you've got a mixture of rgb, hex and plain colours e.g. 'black'

**Review by daniellaskinner — APPROVED** [review]
see comments

File: index.php (hunk 3 above)
Comment (ChrisTregaskis) [reply index.php:9]: Great thanks! took it out and it all still worked fine

**Pull request merged**